packages/openapi-ts/src/plugins/fix-schema/config.ts

```diff
@@ -0,0 +1,18 @@
+import type { Plugin } from '../types';
+import { handler } from './plugin';
+import type { Config } from './types';
+
+export const defaultConfig: Plugin.Config<Config> = {
+  _dependencies: [],
+  _handler: handler,
+  _handlerLegacy: () => {},
+  name: 'fix-schema',
```

Comment:
I'm assuming you built this because you have an actual use case. Is it a common use case for you? Or it happened only with one specification and you've decided to build this plugin?

Reply:
That's why I added a note in the guide documentation to specify this plugin's order at the very front. But as you suggested, if we handle it directly in the input, we wouldn't need such precautions. I'll try implementing it that way.